build-aux/snap/snapcraft.yaml

```diff
@@ -10,9 +10,9 @@ description: |
 version: set-by-version-script-thxbye
 version-script: |
   ./mkversion.sh --output-only
-#FIXME: enable once snapcraft understands this
-#type: snapd
+type: snapd
 grade: stable
+base: core
```

Comment:
ideally this should be a build-base instead as we know

Reply:
Done now.